### ISSUE TITLE
Use absolute import in utils and scratch

### ISSUE THREAD

**Issue by sahmed95** [nipy/dipy#1002]
This is the last issue regarding absolute imports in dipy. Please refer to the original issue #969. Use absolute import in place of relative for all .py files inside /dipy. Take care not to change .pyx files or the __init__ files

```python
/dipy/dipy/utils/optpkg.py:
   13      have_nose = True
   14  
   15: from .tripwire import TripWire
   16  
   17  if have_nose:

/dipy/dipy/utils/tests/test_arrfuncs.py:
    6  import numpy as np
    7  
    8: from ..arrfuncs import as_native_array, pinv, eigh
    9  
   10  from numpy.testing import (assert_array_almost_equal,

/dipy/dipy/utils/tests/test_tripwire.py:
    2  """
    3  
    4: from ..tripwire import TripWire, is_tripwire, TripWireError
    5  
    6  from nose import SkipTest

/dipy/scratch/learning_old.py:
    2  
    3  import numpy as np
    4: from . import track_metrics as tm
    5  import dipy.core.track_performance as pf
    6  from scipy import ndimage as nd


```

**Comment by theaverageguy** [comment]
I am working on this one.

**Comment by sahmed95** [comment]
@theaverageguy I guess @souravsingh is already working on this. I will change the issue to take care of only utils and scratch to avoid conflicts with @souravsingh 's PR.

**Comment by theaverageguy** [comment]
No problem. :) Will take up the other half @souravsingh ! Thanks

**Comment by arokem** [comment]
I believe this was resolved in #1006
